### PR TITLE
Update unidirectional immutable example

### PR DESCRIPTION
**Pull request by joshgillies**
First attempt at using Facebook's `immutable` library in place of `observ-*`.

Ping #75

### CODE REVIEW

File: examples/unidirectional/immutable/browser.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+function toObserv(cursor) {
+    return function observ(listener) {
+        if (!listener) {
+            return cursor.deref().toJS();
+        }
+
+        cursor._onChange = function onChange(data) {
+            cursor._rootData = data;
+            listener(data.toJS());
```
Comment (Raynos) [inline examples/unidirectional/immutable/browser.js]: I'm afraid this line ruins the performance. Doing a deep copy of an immutable data structure into a plain mutable javascript object defeats the point of doing persistent data structures.

**Comment by Raynos** [comment]
lgtm. Maybe pass in an immutable object instead of calling `.toJS()`

**Comment by joshgillies** [comment]
Thanks, I too noted this right after submitting the PR. :(

As such I already have a cut of what your proposing locally, will push
these changes soon.

**Comment by joshgillies** [comment]
Updated. Take a look and let me know if there's anything else.

**Comment by Raynos** [comment]
@joshgillies looks legit. #nicework

**Pull request merged**